ruby/command-t/lib/command-t/scanner/file_scanner.rb

```diff
@@ -10,11 +10,27 @@ class Scanner
     # RubyFileScanner and FindFileScanner subclasses).
     class FileScanner < Scanner
       # Subclasses
+      autoload :CommandFileScanner,  'command-t/scanner/file_scanner/command_file_scanner'
       autoload :FindFileScanner,     'command-t/scanner/file_scanner/find_file_scanner'
       autoload :GitFileScanner,      'command-t/scanner/file_scanner/git_file_scanner'
       autoload :RubyFileScanner,     'command-t/scanner/file_scanner/ruby_file_scanner'
       autoload :WatchmanFileScanner, 'command-t/scanner/file_scanner/watchman_file_scanner'
 
+      def self.for_string s
```

Comment:
This is fine. We could use a more descriptive word than "string" though. Maybe `for_scanner_type` or something like that.